crates/dreammaker/tests/constants_tests.rs

```diff
@@ -110,3 +110,11 @@ fn rgb_hcy() {
         Constant::string("#000000"),
     );
 }
+
+#[test]
+fn fexists_noarg() {
+    assert_eq!(
+        eval("fexists()").unwrap_err().description(),
+        "malformed fexists() call, must have 1 argument and instead has 0",
+    );
+}
```

Comment:
True/false tests would be good here too.

Reply:
Couldn't get tests working, due to either not evaluating the preprocessor or the ability to create files within the tests.

Reply:
guh, those didn't work when i tried, oh well ty